addons/payment/data/ir_actions_server_data.xml

```diff
@@ -3,12 +3,10 @@
 
     <record id="action_activate_stripe" model="ir.actions.server">
         <field name="name">Activate Stripe</field>
-        <field name="model_id" ref="website_payment.model_payment_provider"/>
+        <field name="model_id" ref="payment.model_payment_provider"/>
         <field name="state">code</field>
         <field name="code">
-menu = env.ref('website.menu_website_website_settings', raise_if_not_found=False)
-menu_id = menu and menu.id
-action = env.company._run_payment_onboarding_step(menu_id=menu_id)
+action = env.company._run_payment_onboarding_step()
```

Comment:
Add and explain why this change in the task too please

Reply:
I mentioned it in dev notes and for convenience duplicated it here.
As an action `action_activate_stripe​` should be available for all modules that call `payment.form​` (because we move payment warning to the form) the menu of website setting was removed to allow it to fall back to the default one.